allennlp/data/dataset_readers/sequence_tagging.py

```diff
@@ -27,9 +27,9 @@ class SequenceTaggingDatasetReader(DatasetReader):
 
     Parameters
     ----------
-    word_tag_delimiter : ``str``, optional (default=``"###"``)
+    word_tag_delimiter: ``str``, optional (default=``"###"``)
```

Comment:
Not sure what this one is, but it's probably the backticks in the `default=` part.